client/consensus/babe/rpc/Cargo.toml

```
@@ -31,6 +31,7 @@ sp-application-crypto = { version = "2.0.0-rc6", path = "../../../../primitives/
 sc-keystore = { version = "2.0.0-rc6", path = "../../../keystore" }
 
 [dev-dependencies]
+parking_lot = "0.10.2"
```

```suggestion
parking_lot = "0.10.0"
```

Tiny nit: While I prefer using a more up-to-date version, I would prefer (a) this to happen in a separate pull request and (b) across the entire repository.